app/controllers/api/v2/UsersController.js

```diff
@@ -127,6 +127,19 @@ export default class UsersController {
       ]);
       const allGroupAdmins = await dbAdapter.getGroupsAdministratorsIds(_.map(_.filter(allUsers, { type: 'group' }), 'id'), user.id);
 
+      {
```

Comment:
@davidmz why this kind of '{}' is used here?

Reply:
I use it in some places for variable isolation. Although it is not strictly needed here, it guarantees that additionalUIDs will not be visible outside of {}.